GOVERNANCE.md

```diff
@@ -54,6 +54,11 @@ If a vote is taken during a WG meeting, the follow rules will be followed:
   [here](https://docs.google.com/spreadsheets/d/1bw5s9sC2ggYyAiGJHEk7xm-q2KG6jyrfBy69ifkdmt0/edit#gid=0).
   Members must acknowledge their presence verbally, meaning, adding yourself
   to the "Attendees" section of the Agenda document is not sufficient.
+* When a vote is called, votes will be collected over the following week: 
```

Comment:
I'd prefer if we didn't mandate a week since its possible all voting members are on the call when the vote is started.  So what about:
* When a vote is called if all voting members are present in the meeting, and the all agree to vote at that time, then votes will be tallied during that meeting.
* If not all voting members are present, or some wish more time, then those members will be allowed one week to vote asynchronously. In the case of a PR vote, they will post a comment to the PR with their vote. In other cases an email to the CloudEvents mailing list is expected.
* In the case of yes/no votes, each vote will be either "yes", "no" or "abstain". In the case of non-yes/no votes, the WG will agree on the valid responses prior to starting the vote.